main.py

```diff
@@ -809,7 +813,7 @@ def create_years_list():
         try:
             year_col = db.session.query(distinct(func.extract("year", Marker.created)))
             break
-        except OperationalError:
+        except:
```

Comment:
OperationalError is a generic MySQL exception (_mysql_exceptions). May be, it would be better to replace it by SQLAlchemy exceptions?